utils/docker/Dockerfile.centos.7

```diff
@@ -65,52 +118,37 @@ RUN yum clean all && \
         python36-Cython \
         python36-devel \
         python36-distro \
-        python36-jira \
         python36-numpy \
         python36-paramiko \
         python36-pylint \
         python36-requests \
-        python36-requests \
         python36-tabulate \
         python36-pyxattr \
+        python36-PyYAML \
         python36-scons \
         sg3_utils \
         valgrind-devel \
         yasm && \
-    yum clean all
+    dnf clean all
 
 # Install this individually because of potential conflicts
-RUN yum -y install openmpi3-devel && yum clean all
+RUN dnf -y install openmpi3-devel && dnf clean all
```

Comment:
The comment and separate install of this command doesn't make sense.  If it were going to conflict being installed in the previous transaction it would also conflict here.  There is nothing between these two transactions that would make them work separately any differently than they would in a single transaction.

Reply:
Fixed in next push.